packages/extension-ui/src/Popup/Signing/Request.tsx

```diff
@@ -3,37 +3,43 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { ExtrinsicPayload } from '@polkadot/types/interfaces';
-import { AccountJson, RequestExtrinsicSign } from '@polkadot/extension/background/types';
+import { AccountJson, RequestSign } from '@polkadot/extension/background/types';
 
 import React, { useContext, useState, useEffect } from 'react';
 import { createType } from '@polkadot/types';
 
 import { ActionBar, ActionContext, Address, Link } from '../../components';
 import { approveSignPassword, approveSignSignature, cancelSignRequest } from '../../messaging';
-import Details from './Details';
+import Extrinsic from './Extrinsic';
+import Bytes from './Bytes';
 import Qr from './Qr';
 import Unlock from './Unlock';
+import { SignerPayloadJSON, SignerPayloadRaw } from '@polkadot/types/types';
```

Comment:
(small nitpick) Move this to the "types section", i.e. the first bundle of imports - alphabetical, so will end up on line 7 with line 8 being the new blank.